src/extensions/codemirror/hooks/useCodemirrorModeExtension.hooks.ts

```diff
@@ -32,13 +31,22 @@ const supportedModes = {
     javascript,
 } as const;
 
+const v6AdaptedModes = new Map([
```

Comment:
Please reuse the definitions from `SupportedCodeEditorModes`, look for a very similar example here: https://github.com/eccenca/gui-elements/pull/243/files#diff-fd64753b53d2a93bf6dbcd2e8d6fb8dd9fe3412001a035add2c9adca4d2f0a7fR166

Reply:
Reuse?? they both serve different purposes one is the general list of supported modes, the other categorizes the modes that have the newly adaptations. The modes are called differently

Reply:
I asked to reuse the signature for the mapping, `json`, `markdown`, `xml`, `sql`, `yaml`, `javascript` are looking exactly like the type keys from `SupportedCodeEditorModes`. I'll try it myself.